salt/modules/rabbitmq.py

```diff
@@ -828,24 +828,46 @@ def list_policies(vhost="/", runas=None):
         python_shell=False)
     _check_response(res)
     output = res['stdout']
+
+    if __salt__['pkg.version']('rabbitmq-server'):
```

Comment:
Would using something like `salt.utils.versions.LooseVersion` be beneficial here?

Reply:
@rallytime Yes, I took a look at that. I'm thinking about to split up the rabbitmq.py module to a freebsd_rabbitmq.py and a default rabbitmq.py module, because of the different package names. If I do it like this I can set the package name in the freebsd module and don't have to make an ugly if-else construct. At the moment I'm not sure if I should copy all functions into the freebsd_rabbitmq.py module. If I do this we have a lot of duplicate code (freebsd_rabbitmq.py and rabbitmq.py). Do you have an idea how to solve this nicely? 

You're right, I should use LooseVersion to compare the versions.

Reply:
The if/else is fine, but to answer your question, we use the files in the `utils` directory to hold code that is used across multiple modules/states.